api/src/main/java/org/openmrs/Role.java

```diff
@@ -72,11 +127,17 @@ public void setPrivileges(Set<Privilege> privileges) {
 		this.privileges = privileges;
 	}
 	
-	@Override
 	public String getName() {
 		return this.getRole();
 	}
 	
+	/**
+	 * @param name the name to set
+	 */
+	public void setName(String name) {
```

Comment:
What's the point of having this setter and keeping the `name` field around if we neither return the name from `getName()` nor store the field? (I understand keeping `setName()` around, just have it do nothing).